portal/views.py

```diff
@@ -150,7 +131,7 @@ def get_public_apps(request, app_id=0, collection=None, order=None):
 
 
 def public_apps(request, app_id=0):
-    published_apps, request = get_public_apps(request, app_id=app_id)
+    published_apps, request = get_public_apps(request, app_id=app_id, order_by="updated_on", order_reverse="True")
```

Comment:
```suggestion
    published_apps, request = get_public_apps(request, app_id=app_id, order_by="updated_on", order_reverse=True)
```
Shouldn't it be like this?